config/locales/simple_form.en.yml

```diff
@@ -102,7 +102,7 @@ en:
         highlighted: This makes the role publicly visible
         name: Public name of the role, if role is set to be displayed as a badge
         permissions_as_keys: Users with this role will have access to...
-        position: Higher role decides conflict resolution in certain situations
+        position: Higher role decides conflict resolution in certain situations. Moderation actions can only target accounts with a lower priority
```

Comment:
Maybe `Certain actions can only be performed on roles with lower priority`? Thinking about stuff like having the power to delete moderation notes of other people.

Reply:
Works for me. Updated.